source/admin_manual/list_of_events.rst

```diff
@@ -2315,6 +2315,29 @@ indexes in metacache.
 fs-dictmap
 ----------
 
+
+fs_dictmap_dict_write_uncertain
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. versionadded:: 2.3.13
+
+The event is sent whenever a dict write is uncertain.
+
++-----------------------+------------------------------------------------------+
+| Field                 | Description                                          |
++=======================+======================================================+
+| Inherits from fs_file                                                        |
++-----------------------+------------------------------------------------------+
++-----------------------+------------------------------------------------------+
+| path                  | Virtual FS path to the object (based on dict)        |
++-----------------------+------------------------------------------------------+
+| object_id             | Object ID in the storage                             |
++-----------------------+------------------------------------------------------+
+| cleanup               | success, failed or disabled                          |
```

Comment:
Should have a longer explanation, like: If the uncertain write was attempted to be cleaned (deleted), result of the attempt.

Reply:
Maybe also point to the setting that can be used to enable/disable it?